scripts/job_nek5k

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+# Usage:
+# 1. Copy to the directory where you have your files
+# 2. Update any needed environment variables and input file names in this script
+# 3. sbatch job_nek5k
+
+#SBATCH --nodes=1
+#SBATCH --ntasks-per-node=40
+#SBATCH --time=00:20:00
+#SBATCH --output=pincell.log
+#SBATCH -p compute
+
+# Revise for your repository and cross section data locations
+export OPENMC_CROSS_SECTIONS=$HOME/cross_sections/endfb71_hdf5/cross_sections.xml
+export CARDINAL_DIR=$HOME/cardinal
+export NEKRS_HOME=$CARDINAL_DIR/install
+
```

Comment:
```suggestion
# need to point to a newer CMake version
export PATH=/shared/cmake-3.24.2/bin:$PATH
```

Reply:
This isn't actually necessary - the CMake version is only needed during the compile stage, not running.